src/bootstrap.js

```diff
@@ -331,10 +333,11 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
      */
     ChatEngine.subscribeToPubNub = () => {
 
+        // TODO: DIS BISH GOTTA GO
         let chanGroups = [
-            ceConfig.globalChannel + '#' + ChatEngine.me.uuid + '#rooms',
-            ceConfig.globalChannel + '#' + ChatEngine.me.uuid + '#system',
-            ceConfig.globalChannel + '#' + ChatEngine.me.uuid + '#custom'
+            ceConfig.globalChannel + '_' + ChatEngine.me.uuid + '_rooms',
```

Comment:
Did you rewrite the server component too?

Reply:
yes! See the recent merged changes to src/functions/server.js in chat-engine-setup repo.